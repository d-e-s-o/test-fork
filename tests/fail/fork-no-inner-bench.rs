// Copyright (C) 2025 Daniel Mueller <deso@posteo.net>
// SPDX-License-Identifier: (Apache-2.0 OR MIT)

struct Bencher;

#[test_fork::fork]
fn missing_bench(b: &mut Bencher) {}

fn main() {}
