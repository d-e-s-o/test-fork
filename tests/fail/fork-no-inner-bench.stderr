error: test_fork::fork requires an inner #[bench] attribute
 --> tests/fail/fork-no-inner-bench.rs:6:1
  |
6 | #[test_fork::fork]
  | ^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `test_fork::fork` (in Nightly builds, run with -Z macro-backtrace for more info)
